tests/dist/src/test/java/org/hawkular/apm/tests/dist/TraceCompletionTimeITest.java

```diff
@@ -75,10 +75,10 @@ public void initTest() {
     public void testGetCompletionTimesSingleFragment() throws Exception {
         Trace trace1 = new Trace();
         trace1.setId("1");
-        trace1.setStartTime(System.currentTimeMillis() - 4000); // Within last hour
+        trace1.setStartTime(TimeUnit.MILLISECONDS.toMicros(System.currentTimeMillis()) - 4000); // Within last hour
```

Comment:
Same as in previous file (and other places in this file. If fix applied in previous file, then apply here.